model/run.py

```diff
@@ -83,6 +154,7 @@ def compute_KPIs(df: pd.DataFrame):
     df["burn_rate"] = df["POKT_burned"] / df["floating_supply"].shift(1)
     df["mint_rate"] = df["POKT_minted"] / df["floating_supply"].shift(1)
     df["net_mint_rate"] = df["POKT_net_mint"] / df["floating_supply"].shift(1)
+    df["load_balance"] = kpi_c(df, col_name = "servicer_relay_log")
```

Comment:
You end up returning a scalar here which causes all simulations to have the same value for the kpi